apps/dashboard/src/lib/Types.svelte

```diff
@@ -0,0 +1,18 @@
+<!-- TODO: Can we generate this via supabase? -->
+<script lang="ts" context="module">
```

Comment:
- If this just TypeScript, it should be a .ts file.
- Yes all db table types are generated, try this command https://github.com/chandertech/turtlelinks/blob/00375ebc45d8884790bb58e73d08181f15ced15c/apps/dashboard/package.json#L16